meta-rdk-mtk-bpir4/recipes-ccsp/util/gw-lan-refresh_git.bb

```diff
@@ -0,0 +1,12 @@
+LICENSE = "CLOSED"
```

Comment:
LICENSE must not be CLOSED in an open repository (almost always).  Please use "Apache-2.0" .